ceph.spec.in

```diff
@@ -707,6 +707,15 @@ Requires:	libcephfs2 = %{_epoch_prefix}%{version}-%{release}
 %description -n cephfs-mirror
 Daemon for mirroring CephFS snapshots between Ceph clusters.
 
+%package -n ceph-exporter
+Summary: Daemon for exposing perf counters as Prometheus metrics
```

Comment:
apologies for bike-shedding here, but i think a more specific name like `ceph-prometheus-exporter` would be preferable here